applications/MappingApplication/custom_searching/interface_communicator.cpp

```diff
@@ -288,15 +295,27 @@ void InterfaceCommunicator::ConductLocalSearch()
                 }
             }
         }
+
+        if (mEchoLevel > 1) {
+            const auto& r_data_comm = rComm.GetDataCommunicator();
+            sum_num_results = r_data_comm.Sum(sum_num_results, 0);
+            sum_num_searched_objects = r_data_comm.Sum(sum_num_searched_objects, 0);
+
+            const double avg_num_results = sum_num_results / static_cast<double>(sum_num_searched_objects);
+
+            KRATOS_INFO_IF("Mapper", mEchoLevel > 1) << "An average of " << avg_num_results << " objects was found while searching" << std::endl;
+            KRATOS_WARNING_IF("Mapper", avg_num_results > 200) << "Many search results are found, consider adjusting the search settings for improving performance" << std::endl;
+        }
```

Comment:
I realized that this doesn't get executed by all the processes if some partitions are empty
Aka it will just hang in this case :D

interesting that none noticed in 3 Months...

Will fix it today or tmr

probably relevant for you @KratosMultiphysics/altair